frontend/src/metabase/components/LastEditInfoLabel.js

```diff
@@ -45,11 +45,11 @@ function LastEditInfoLabel({ item, user, onClick, ...props }) {
     editorId === user.id ? t`you` : formatEditorName(first_name, last_name);
 
   return (
-    <TextButton
+    <Badge
```

Comment:
The `Badge` component ends up rendering an `<a>` tag which really should only be used for links. Things that are actions like this really ought to be `<button>` tags, even if they are styled to look like links. I added the `TextButton` styled component to try to handle this use case, so can you try continue using `TextButton` and either tweak the css of it or do this:

```
const TextButtonWithAddedStyles = TextButton.extend`
  css-property: foo;
  padding: 1rem;
`

```

And then you can us that as a react component.

Reply:
Cool, thanks for the correction! What I tried to do initially was apply our `text-small` class to that instance of `TextButton` via `className`, but it wasn't seeming to work. I'll see if I can make that work or use that `extend` syntax.